libavif.pc.cmake

```diff
@@ -1,10 +1,10 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-exec_prefix=${prefix}/bin
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
-includedir=${prefix}/include
+exec_prefix=${prefix}
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
```

Comment:
Not much to comment on if these are correct or not (they seem to be, but my experience of using pkg-config is more often using the provided Cflags and Libs rather than writing them), but I do know that nixOS people prefer just using the `_FULL_` version https://github.com/OpenVisualCloud/SVT-HEVC/pull/517. Not sure if it's something that should be considered, as it's somewhat niche.

Reply:
I haven't affected ${prefix} with this change (and the others were already relative, in an incorrect way though), so this is a separate issue.

Reply:
Btw, autotools can automagically replace @exec_prefix@, @libdir@, and @includedir@ without this relative scheme, so there is an option mimic those variables inside CMakeLists.txt, [like libheif does for example](https://github.com/strukturag/libheif/blob/bc5db10bd95c4fa3e66a2028e3c0da0745aff7a2/CMakeLists.txt#L119-L122).